app/verifier/pom.xml

```diff
@@ -365,6 +365,10 @@
         <artifactId>maven-dependency-plugin</artifactId>
         <configuration>
           <ignoredUnusedDeclaredDependencies>
+            <ignoredUnusedDeclaredDependency>io.syndesis:sql-common</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>junit:junit</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>commons-io:commons-io</ignoredUnusedDeclaredDependency>
+            <ignoredUnusedDeclaredDependency>org.assertj:assertj-core</ignoredUnusedDeclaredDependency>
```

Comment:
Sure that this needs to be added here and not by adding the proper scope to dependencies ? The maven dependency plugin does not count "test" and "runtime" scoped dependencies as unused, and e.g. "junit" should be clearly a test scoped dependency.